### ISSUE TITLE
RegExp lookbehinds and Node.js version

### ISSUE THREAD

**Issue by mrmlnc** [micromatch/picomatch#3]
In the documentation you say that in order to use the `lookbehinds` option you need to have a Node.js version greater then `8.1.10`.

* First, there is no such version. I think you mean `8.11.0`.
* This functionality (RegExp lookbehinds) is [available without flags](https://node.green/#ES2018-features--RegExp-Lookbehind-Assertions) only since version 9.
* [Here](https://github.com/micromatch/picomatch/blob/efe4ec9448069352030473b9a3b4e36345b1478d/lib/parse.js#L565-L567) you require a Node.js version 10 or higher.

This is the wrong documentation or check in the code?

**Comment by jonschlinkert** [comment]
> * First, there is no such version. I think you mean `8.11.0`.

thanks! typo, I'll look that up again. 

> This is the wrong documentation or check in the code?

I'm glad you're pointing this out. When I first added support for lookbehinds I did more research on the version, so I'm thinking that whatever version is in the actual code is more likely to be correct. 

I kind of rushed out this release so I missed some things in the docs. Thanks for noticing them. 

I just invited you to the org if you want to make changes. Or I can if you don't have time. I'll be out most of the day today, but I can fix these version issues later when I get back.
